docs/.vitepress/theme/custom.css

```
@@ -107,192 +150,253 @@
+  align-items: center;
+  justify-content: center;
+}
+
+.usage-hero-art {
+  width: 340px;
+  height: auto;
+  animation: artFloat 7s ease-in-out infinite;
+  filter: drop-shadow(0 10px 40px rgba(255, 45, 149, 0.3));
+}
+
+@keyframes artFloat {
```

Stylelint (17.14.0)

[error] Expected keyframe name "artFloat" to be kebab-case (keyframes-name-pattern)

(keyframes-name-pattern)